pkg/registry/storage/csinode/strategy.go

```diff
@@ -68,8 +76,33 @@ func (csiNodeStrategy) AllowCreateOnUpdate() bool {
 	return false
 }
 
-// PrepareForUpdate sets the Status fields which is not allowed to be set by an end user updating a CSINode
+// PrepareForUpdate sets the driver's Allocatable fields that are not allowed to be set by an end user updating a CSINode.
```

Comment:
from the code it seems to clear (reset to nil), instead of set, some allocatable fields.

Reply:
This is a cherry pick from community. Won't fix their comment in such a massive migration. Perhaps we can do it later when we need to revisit this module.